apps/mobile/src/queries/use-market-data.query.ts

```diff
@@ -0,0 +1,42 @@
+import { useSettings } from '@/store/settings/settings';
+import { QueryFunctionContext, useQueries, useQuery } from '@tanstack/react-query';
+
+import { bitcoinNetworkModeToCoreNetworkMode } from '@leather.io/bitcoin';
+import { FungibleCryptoAssetInfo, NetworkModes } from '@leather.io/models';
+import { getMarketDataService } from '@leather.io/services';
+
+export function marketDataQueryOptions(token: FungibleCryptoAssetInfo, network: NetworkModes) {
+  return {
+    queryKey: ['get-market-data', network, token],
+    queryFn: ({ signal }: QueryFunctionContext) =>
+      getMarketDataService().getMarketData(token, signal),
+    refetchOnReconnect: false,
+    refetchOnWindowFocus: false,
+    refetchOnMount: false,
+    retryOnMount: false,
+    staleTime: 1000 * 60,
+    gcTime: 1000 * 60,
```

Comment:
Maybe we can use const's for the `gcTime` and `staleTime` and add some defensive code, maybe a test that will fail if it gets set to an invalid value like Kyran found yesterday? 

https://github.com/leather-io/mono/pull/582/files